packages/instructor-stream/src/utils/token-parser.ts

```
@@ -200,130 +198,127 @@ export default class TokenParser {
-      throw new TokenParserError(
+    return this.error(
+      new TokenParserError(
         `Unexpected ${TokenType[token]} (${JSON.stringify(
           value
         )}) in state ${TokenParserStateToString(this.state)}`
       )
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    } catch (err: any) {
-      this.error(err)
-    }
+    )
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)